### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by LuizDMM** [MongoEngine/django-mongoengine#174]
Hey guys, how are you doing today?

I'm working with a start-up that uses Django and MongoDB, and we're currently using "Djongo" as Object-Document-Mapper, but that project seems to be using the contributions only to their paid version. The pip version hasn't received upgrades for the last couple years, besides having a lot of pushes in main.

That said, I'm looking for a solution to migrate our ODM in the next couple of months. One solution would be to clone their repo and create a new pip package, but I never did that. That said, I have some experience with Django and I can try to help a lot in my free time because it's a core future for us. But before committing to making PRs, I have to ask, is this project alive? If so, how can I help?

**Comment by last-partizan** [comment]
If you're already using Djongo and it works fine for you, i suggest continue using it.

You can install git version with pip easily. Or maybe switch to their paid version, if they're taking money, i suggest they're have time to support it properly.

This project does not receive any active updates, it's in mainternance mode and there will be no new features. But, if you're willing to fix some bugs - i will review PR's and merge them.

**Comment by last-partizan** [comment]
@ngocngoan for your case, i can suggest using raw mongoengine? Do you think you can use it?

**Comment by ngocngoan** [comment]
@last-partizan Yes, I can use it. But, my library is using the MongoUser document as the reference field. So, I need to replace it before removing this dependency
